tests/context_test.py

```diff
@@ -45,3 +52,164 @@ def test_ogp_context():
 
         for inp,out in res:
             assert out == getattr(context,inp)
+
+
+def test_mf2_context(app):
+    """ Check that we can get Microformats2 data from a document
+    """
+    test_input = [
+        '', # empty test
+        '\n'.join([
+            '<div class="h-entry">',
+            '<h1 class="p-name">Test Article</h1>',
+            '<p class="e-content">Words words words</p>',
+            '</div>',
+        ]),
```

Comment:
any reason not to do these as multiline strings (i.e. """...""""? it seems like that would make life a little easier!